packages/@coorpacademy-components/src/organism/mooc-header/style.css

```
@@ -501,6 +501,52 @@
   color: dark;
 }
 
+.homeIcon {
+  width: 16px;
+  height: 14px;
+  margin-left: 16px;
+}
+
+.toolTipText {
+  font-family: 'Gilroy';
+  font-size: 14px;
+  width: 121px;
+  height: 24px;
+  line-height: 27px;
+  order: 0;
+}
+
+[role="tooltip"],
+.tooltipIconContainer + [role="tooltip"] {
```

```suggestion
.logo .tooltipIconContainer + [role="tooltip"] {
```
pour éviter un possible clash avec autres tooltips